prov/efa/src/dgram/efa_ep.c

```diff
@@ -642,10 +436,14 @@ int efa_ep_open(struct fid_domain *domain_fid, struct fi_info *user_info,
 			return ret;
 	}
 
-	ep = efa_ep_alloc(user_info);
+	ep = calloc(1, sizeof(*ep));
 	if (!ep)
 		return -FI_ENOMEM;
 
+	ret = efa_base_ep_open(&ep->base_ep, user_info);
```

Comment:
should be `construct` because you are not allocating memory for `ep->base_ep`. See `naming_convention.md` under docs.

Reply:
Fixed